vsintegration/packages.config

```diff
@@ -8,12 +8,13 @@
   <package id="System.Threading.Tasks.Dataflow" version="4.5.24" targetFramework="net46" />
   <package id="Microsoft.Composition" version="1.0.30" targetFramework="net46" />
 
-  <package id="Microsoft.CodeAnalysis.Common" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.EditorFeatures" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.EditorFeatures.Text" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.Features" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="2.3.0-beta2-61719-01" targetFramework="net46" />
-  <package id="Microsoft.VisualStudio.LanguageServices" version="2.3.0-beta2-61719-01" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Common" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures.Text" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.EditorFeatures.Wpf" version="2.6.0-vs-for-mac-62329-05" targetFramework="net46" />
```

Comment:
@jasonmalinowski :

Hi Jason, is the EditorFeatures.Wpf package still relevant in any way?
I am in the process of darc-ifiying our Roslyn dependencies (to spot issues earlier than when things are bound in VS) and I noticed this `Microsoft.CodeAnalysis.EditorFeatures.Wpf` has not been updated in years.

Which lead to me to this PR which added it despite not being used in the code....